workspace/python/ds-pack/2.0.0-base-python-3.11.2/README.md

```diff
@@ -0,0 +1,68 @@
+# workspace-python-ds-pack-2.0.0-base-python-3.11.2
+This is the Python Data Science pack image made for Workspaces. 
+The image runs a [Jupyterhub](https://jupyter.org/hub) server that has [VSCode](https://github.com/coder/code-server) pre-installed with all of the essential and most used packages.
+Note: It doesn't support [PyCharm](https://lp.jetbrains.com/projector/) unlike previous Python workspace image versions.
+
+## Image details
+### Base image
+This image uses [python:3.11.2-slim-bullseye](https://hub.docker.com/layers/library/python/3.11.2-slim-bullseye/images/sha256-bb315a212e098bf2a95d9b444f0eee78b5dffb96caac78ac5fddf2ce3134adf1?context=explore) as its base which is maintained by [the Docker Community](https://github.com/docker-library/python).
+
+### OS and other details
+```
+Debian         GNU/Linux 11 (bullseye) (11.6-slim-bullseye)
+Linux Kernel   5.10.176
+Python         3.11.2
+```
+
+### Important Linux packages installed
+```
+aws-cli        2.7.20
+curl           7.74.0
+git            2.30.2
+jq             1.6-2.1
```

Comment:
I get only `1.6` when I run `jq --help`.

Reply:
For nano, its `5.4`

Reply:
jq/now 1.6-2.1 amd64 [installed,local]
nano/now 5.4-2+deb11u2 amd64 [installed,local]

Can see similar patterns for vim, pandoc, etc in Image build logs:  `vim            8.2.2434-3`

Reply:
Ahh, I tried running directly on the workspace 😅